src/smart-components/group/add-group/add-group-wizard.js

```diff
@@ -57,9 +60,13 @@ const AddGroupWizard = ({
       user_list: selectedUsers ? selectedUsers.map(user => ({ username: user.label })) : undefined,
       roles_list: selectedRoles ? selectedRoles.map(role => role.uuid) : undefined
     };
-    await addGroup(user_data);
-    postMethod();
-    history.push(closeUrl);
+    try {
+      await addGroup(user_data);
+      postMethod();
+      history.push(closeUrl);
+    } catch (e) {
+      setIsGroupInfoValid(false);
```

Comment:
Does api return what kind of error happened? Can we differentiate the name is taken error and other errors?